etsin_finder/frontend/js/components/qvain/index.jsx

```diff
@@ -144,8 +157,12 @@ class Qvain extends Component {
   }
 
   handleCreate = e => {
-    e.preventDefault()
-    this.setState({ submitted: true })
+    // e.preventDefault()
```

Comment:
`preventDefault()` is used to prevent the whole page from re-rendering. Is there a particular reason why it is commented out?

Reply:
Good point. I don't think there is a good reason, apart from making things work quickly. The initial reason was that e.preventDefault() prevented the publishing from happening after clicking "yes" in the DOI accept window, since the event sent to handleCreate was a different one, coming from the modal window.

Now enabled e.preventDefault() again, taking care of the DOI window event with if logic in handleCreate(): should be resolved with commit b4eb52a